omnibus/README.md

```diff
@@ -10,7 +10,8 @@ This project creates full-stack platform-specific packages for the following pro
 You must have a sane Ruby environment with Bundler installed. Ensure all the required gems are installed:
 
 ```shell
-bundle install --without development
+bundle config set without development
```

Comment:
The instructions in this file all need to be the local version of the bundle config command so that users don't set a global without.

Reply:
Sure, I'll update that accordingly.